### PR TITLE
Components

### PR DESCRIPTION
**Pull request by men32z**
- App
- Button
- ButtonPanel
- Display
- Basic structure for components.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    {name}
+  </div>
+);
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: 'Button',
+};
```
Comment (shloch) [inline src/components/Button.js]: is this necessary ??? 
if it is, it should be an empty string `' '`

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const group1 = ['AC', '+/-', '%', '+'].map(x => <Button key={x} name={x} />);
+  const group2 = ['7', '8', '9', 'X'].map(x => <Button key={x} name={x} />);
+  const group3 = ['4', '5', '6', '-'].map(x => <Button key={x} name={x} />);
+  const group4 = ['1', '2', '3', '+'].map(x => <Button key={x} name={x} />);
+  const group5 = ['0', '.', '='].map(x => <Button key={x} name={x} />);
```
Comment (shloch) [inline src/components/ButtonPanel.js:9]: this is wise move

**Review by shloch — APPROVED** [review]
## Hello @men32z 
Great job...
You can merge, you code is clean

(reason with me a little for below code :) )

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (men32z) [reply src/components/ButtonPanel.js:9]: thanks!

File: src/components/Button.js (hunk 1 above)
Comment (men32z) [reply src/components/Button.js]: No, my mistake, I read again the requirements and it's no necessary to set default props in this component, thanks!

**Pull request merged**